flask_praetorian/constants.py

```diff
@@ -32,10 +41,12 @@
 
 REFRESH_EXPIRATION_CLAIM = 'rf_exp'
 IS_REGISTRATION_TOKEN_CLAIM = 'is_ert'
+IS_RESET_TOKEN_CLAIM = 'is_ert'
```

Comment:
seems like the reset token claim key should be different than the registration token claim key

Reply:
Good catch ... fixed